scripts/zipformer-transducer/reazonspeech/run.sh

```
@@ -0,0 +1,93 @@
+echo "copy onnx model files"
+
+src=/sherpa-onnx/model-files
+mkdir -p $src
+
+mv -v $dir/encoder-epoch-99-avg-1.onnx $src/encoder.onnx
+mv -v $dir/decoder-epoch-99-avg-1.onnx $src/decoder.onnx
+mv -v $dir/joiner-epoch-99-avg-1.onnx $src/joiner.onnx
+
+cp -v $dir/tokens.txt $src
+cp -av $dir/test_wavs $src
+rm -rf $dir/
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)